jep/0000/README.adoc

```diff
@@ -0,0 +1,243 @@
+Jenkins Ambassadors are volunteer representatives (not employees), leaders in the Jenkins community committed to sharing their Jenkins expertise.
+No matter how they contribute -
+whether they are blogging, writing books, speaking, creating tutorials and classes,
+offering support in forums, or organizing and contributing to JAMs and other events –
+they help make Jenkins better.
+
+
+=== Benefits to Ambassador
+
+* Recognition via jenkins.io
+* “Ambassador” blog highlighting their accomplishments
+* Free ticket to Jenkins World, JUC Israel
+* Staff Ask the Expert booth @ Jenkins World
+* Free access to certifications
+* Special Ambassador gift
+
+=== Responsiblities of Ambassadors
+A Jenkins Ambassador responsibility include any of the following:
+
+* Minimum requirements:
+** Author two (2) Jenkins blogs on jenkins.io during the one year term
+** Host at least 2 Jenkins meetups
+* Provide feedback, ideas, and direction that will help to improve the Jenkins project.
+  This involvement may be in the form of technical discussions, contributor summits,
+  and or community building events/activities.
+* Write blog(s) on jenkins.io, speaking engagements at local events, run workshops,
+  develop or contribute to the development of tutorials, participate in
+  and/or organize local events.
+* Staff Jenkins World Ask the Expert booth and booths at other events organized
+  by Jenkins project, is FOSDEM, ScaLE (if local)
+
+
+=== Term
+
+* This is an annual position
+
+
+
+
+=== How are Jenkins Ambassador selected?
+
+The Jenkins Ambassador program is Jenkins Community program sponsored by CloudBees.
+However, the program is not a CloudBees promotional program.
+Care will be taken to avoid CloudBees favoritism or even the appearance of such.
+The program will be largely self-governing with members making most decisions.
+CloudBees employees may be Ambassadors.
+There is no set limit to the number or percentage of CB employees that can be Ambassadors (however, see avoiding favoritism above).
+
+Jenkins Ambassadors operate as an independent group who use a consensus review process to select new members.
+All communication and discussion are conducted via the TBD (a public) mailing list.
+Jenkins Ambassadors receive via an email, the nominee's information based on the criteria above.
+Voting to select the nominee happens via the mailing list with either +1 or -1 from the peer group over a two-week period.
+If the nominee receives three +1 votes and zero negative votes by the end of two weeks then that person is confirmed as a Jenkins Ambassador.
+If there is a -1 vote then this triggers a discussion after which the negative vote may or may be changed. If the negative vote remains, the nominee cannot be confirmed.
+If there are less than three +1 votes during the two-week voting period then further discussion needs to be encouraged.
+
+If there are less than three Ambassadors, the following Jenkins project contributors may cast votes:
```

Comment:
I would rather prefer a generic term like "Jenkins Board members and officers". Do not see why JEP would need to specify particular names if it is considered as a community program